scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala

```diff
@@ -1673,6 +1673,16 @@ object InternalDistriOptimizerV2 {
   }
 }
 
+object Models {
+  def loadModel[T: ClassTag](path: String)(implicit ev: TensorNumeric[T]): keras.Model[T] = {
```

Comment:
how about keras.Sequential?

Reply:
Fixed in https://github.com/intel-analytics/BigDL/pull/3832